en/docs/services/create-your-first-service.md

```diff
@@ -1,168 +1,161 @@
 # Create Your First Service
 
-To learn how to create a service, let's try a simple scenario. In this example, you are creating a service that consumes a public API that fetches the current active COVID-19 cases in a specified country, testing it, deploying it, and then observing its performance.
+To learn how to create a service, let's look at a simple scenario. In this example, you'll create a service that consumes a public API that fetches the current active COVID-19 cases in the United States. Then you'll test it, deploy it, and observe its performance.
 
 ## Step 1: Create the service
 
-To create the service, follow the procedure below: 
+To create the service, follow this procedure: 
 
-1. Access the Choreo Console via https://console.choreo.dev/.
+1. Access the Choreo Console via https://console.choreo.dev/ and sign in using your Google or GitHub credentials.
     
-2. Sign in using either your Google or GitHub credentials.
-    
-3. In the **Services** page, click **Create**. You are directed to the low-code view.
+2. On the **Services** page, click **Create**. You are directed to the **Create Service** page.
 
-4. Under **Create with Choreo**, enter `covid-stats` as the name of your service.
+3. Under **Create with Choreo**, enter `covid-stats` as the name of your service.
```

Comment:
shouldn't the user input be in bold? https://docs.microsoft.com/en-us/style-guide/procedures-instructions/formatting-text-in-instructions

Reply:
The format used in the create api page is a little different. For e.g. in that we have mentioned the field in bold, and then the value. Should we follow a consistent way of presenting the inputs?

Reply:
Although the copy editor suggested bold to format user input values, we want to suggest the use of backticks because it can be confusing when the field names and user input values are formatted in the same way.